masq_lib/src/test_utils/mock_websockets_server.rs

```diff
@@ -490,120 +532,205 @@ mod tests {
     }
 
     #[test]
-    fn broadcast_trigger_can_work_together_with_conversational_messages() {
-        let port = find_free_port();
-        let expected_ui_setup_broadcast = UiSetupBroadcast {
-            running: false,
-            values: vec![UiSetupResponseValue {
-                name: "direction".to_string(),
-                value: "to UI".to_string(),
-                status: Set,
-            }],
-            errors: vec![],
-        };
-        let expected_ui_setup_response = UiSetupResponse {
-            running: true,
-            values: vec![],
-            errors: vec![],
+    fn conversational_and_broadcast_messages_can_work_together_testing_corner_cases() {
```

Comment:
If this test is going to be depended on to (among all the other things it does) take the place of `two_in_two_out()` above, it should check the guts of at least one request and at least one response. On the other hand, if you're happy with the responsibilities of this test as they stand, then leave the happy-path parts of `two_in_two_out()` above.